kcp-dev/kcp
# :book: use hugo modules to fetch dependencies

## Summary

- move all docs dependencies to another repo
- fetch docs dependencies with hugo modules
- remove theme submodule and use hugo modules to fetch theme
- move common config options to another repo

## How to test?

- clear any previous hugo/node cache `hugo mod clean --all`
- launch `hugo serve`
- check the deployment for issues

## References

- kcp-dev/kcp.io#19
